### PR TITLE
Feat:- Input and text-utilities component

### PR DESCRIPTION
**Pull request by anurag-kkumarr**
Added Input component
   -textbox
   -with validation and errors style
  
Text-utilities added
    -headings
    -small text
    -gray text
   -center text

### CODE REVIEW

File: components/CSS/card.css [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    display: flex;
+    font-family: --font-playfair;
+    flex-direction: column;
+    padding: var(--size-one) 8px;
+    border: 1px solid gray;
+    align-items: center;
+    box-shadow: 0 0 5px 5px lightgray;
+  }
+  .card-dimension {
+    width: 500px;
```
Comment (harshitpaliwal95) [inline components/CSS/card.css:20]: use rem as units

**Review by harshitpaliwal95 — CHANGES_REQUESTED** [review]
- use rems as unit otherwise if will be tough to add responsiveness

**Review by Balakumar3018 — APPROVED** [review]
Looks good to me

**Review by masraniJay — APPROVED** [review]
Looks good.

**Pull request merged**